### DISCUSSION TITLE
Source MySQL creation problems

### DISCUSSION THREAD

**Discussion by imartinezcascallana** [airbytehq/airbyte#37097] [Connector Questions]
Hello!

I have created a local database using the MAMP application.

From MySQL Workbench I can access without problems, but from Airbyte I get the following error:
"Internal message: io.airbyte.commons.exceptions.ConnectionErrorException: java.sql.SQLTransientConnectionException: HikariPool-1 - Connection is not available, request timed out after 60002ms (total=0, active=0, idle=0, waiting=0 )"

Someone knows what can it be?

Thanks in advance,
All the best

**Comment by marcosmarxm** [comment]
Is the Airbyte server in the same machine? Maybe try different IPs to access the database or make sure you included in the Docker network (if you're using docker deployment)

**Reply by imartinezcascallana** [reply]
I'm using docker desktop, and I have Airbyte installed locally. How could I check if it is included in the docker network?

**Reply by imartinezcascallana** [reply]
I'm also trying the Airbyte free trial in the browser just in case, and I'm having the same error.

[a comment by an automated account is not shown]
